lib/splash.cjs

```diff
@@ -0,0 +1,43 @@
+const { remove, mkdirp } = require('fs-extra')
+const { join } = require('pathe')
+const sharp = require('sharp')
+
+const devices = [
```

Comment:
@kevinmarrec 
I want move this into CONSTANT file with 

name,
imageWidth,
imageHeight,
mediaQuerieWidth,
mediaQuerieHeight

to get ONE source of thruth

to generate , then inject meta

Reply:
Indeed, we need a file that stores an array of objects containing each device configuration